blueocean-dashboard/src/main/js/PipelineRoutes.jsx

```diff
@@ -127,6 +125,27 @@ function onLeaveCheckBackground() {
 
 const trends = AppConfig.isFeatureEnabled('trends');
 
+function getActions(extensionPoint, params, callback) {
+    Extensions.store.getExtensions([ extensionPoint ], Extensions.Utils.sortByOrdinal, (actions = []) => {
```

Comment:
Search the registered extensions of the given extension point for a matching action identified by name. If an action is found, call the callback on the associated component. If no match is found, call the callback on the `NotFound` component.